src/styles/base/index.scss

```diff
@@ -14,6 +14,30 @@ a:hover {
   padding: 34px 0 34px 0;
 }
 
+h1 {
+  font-size: 60px;
+  font-weight: 800;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.2;
+  letter-spacing: 6px;
+  text-align: left;
+  color: #000000;
+  text-align: left;
+}
+
+p  {
+  font-size: 16.5px;
```

Comment:
don't use decimals numbers with `px` measure. Also please specify `font-size: 16px` for your `:root` in base styles, and then use relatives points, like `rem` or `em` (it is good to know what you can use besides px)